app/views/users/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="col-md-9">
+  <div class="profile-head">
+    <br>
+      <ul class="nav nav-tabs" id="myTab" role="tablist">
```

Comment:
id="myTab" ?
Remove id if not needed. If needed, have descriptive id name and follow underscore convention.

Reply:
Removed